Jamroot.jam

```diff
@@ -0,0 +1,20 @@
+path-constant LIBTORRENT_ROOT : . ;
```

Comment:
I tried putting it in Jamfile itself but it wouldn't pick it up. Apparently I can reference targets from another Jamfile but not rules.

I also tried making asio_src into a static library, so I can pick it up as a pre-made target. That resulted in the -fPIC building errors, because apparently all the build flags have to match up exactly. Just adding source file to an exe target that already has the right flags seemed like the more robust way to go, hence the rule instead of a `lib asio_src`.

and as mentioned, to use the rule from everywhere I apparently have to put it into the Jamroot.jam.